src/lib/config.ts

```diff
@@ -1366,6 +1366,7 @@ export async function update() {
                         // Keep only the ones with im_* functions
                         .filter(
                             key =>
+                                mapobj[key] &&
                                 mapobj[key].search(
```

Comment:
A simpler way of doing this is `mapobj[key]?.search(` - called optional chaining, recently introduced new feature.

Reply:
Optional chaining is a Firefox74 feature, so it won't be available for ESR for a while. Are Tridactyl's source processed by babel?